dev.fredol.open-tv.yaml

```diff
@@ -0,0 +1,1152 @@
+id: dev.fredol.open-tv
```

Comment:
there is no webpage at https://fredol.dev/ can you put something up to confirm that you own the domain ? A link to this PR or the github repo of the app is fine.

Reply:
Fixed, you can retry accessing the domain

Reply:
thanks